src/subgraph/concatenate.c

```diff
@@ -408,18 +304,24 @@ static enum xnn_status check_datatype_copyable(
   return xnn_subgraph_check_quantization_parameter_matches(node_type, input_id, input_value, output_id, output_value);
 }
 
-enum xnn_status xnn_define_concatenate_n(
+static enum xnn_status setup_concatenaten_operator(
+  const struct xnn_operator_data* opdata,
+  const struct xnn_value* values,
+  size_t num_values,
+  pthreadpool_t threadpool)
+{
+  return setup_concatenate_n_operator(opdata, values, num_values, opdata->num_inputs, threadpool);
```

Comment:
Inline this function here. We don't want layers of functions that do trivial work like unwrapping one extra argument.